src/Backend/Modules/ContentBlocks/Domain/ContentBlock/ContentBlockDataGrid.php

```diff
@@ -23,7 +23,7 @@ public function __construct(Locale $locale)
              WHERE i.status = :active AND i.language = :language',
             ['active' => Status::active(), 'language' => $locale]
         );
-
+        $this->setColumnFunction('htmlspecialchars', ['[title]'], 'title', false);
```

Comment:
Hmm seems easy to forget, I wonder if it can automatically apply this function somehow 👀

Reply:
it is indeed, but we also have fields that have html in it. And you only need it on fields that have user input.
Do you have a suggestion on how to make this easier?
We could always add an extra method of colums that you can pass that need to be escaped and then run this internally?

Reply:
Yeah not sure, I'd leave it like this for now given it's a security fix. 
Maybe in future `htmlspecialchars` could be applied by default on datagrids and have a `raw` function you can apply on the few html-safe columns. Like twig has as a `raw` modifier 👀

Reply:
it would probably be safer to change the datagrid implementation in the next mayor away from spoon :p